src/services/PolygonIOStockPriceService.ts

```diff
@@ -16,31 +16,33 @@ class PolygonIOStockPriceService {
   @inject(PolygonIOSingleStockPriceFetcher) polygonIOSingleStockPriceFetcher!: PolygonIOSingleStockPriceFetcher;
   @inject('Settings') settings!: Settings;
   @inject(TimeService) timeService!: TimeService;
-  @inject(PriceAggregator) priceAggregator!: PriceAggregator;
+  @inject(PriceRepository) priceRepository!: PriceRepository;
 
-  static readonly Prefix = 'pios::';
+  static readonly Prefix = 'EQ:';
```

Comment:
why `EQ:`?

Reply:
I don't see PolygonIOStockPriceService on feeds file 🤔 maybe we are not using it
Also, IIRC it was because the symbol was just EQ:symbol, but we filled on feeds file only the `symbol` without EQ: